docker/jdk9/x86_64/ubuntu/Dockerfile-openj9

```diff
@@ -20,8 +20,18 @@ RUN apt-get update \
   && apt-get install -qq -y --no-install-recommends \
     autoconf \
     libnuma-dev \
+    g++-4.8 \
```

Comment:
minor: `gcc-4.8` and `g++-4.8`could be replaced with constants or labels - makes it easier to manage versioning of dependencies

Reply:
@neomatrix369  sorry, I'm not quite sure what you mean here but, ignoring that, it sounds like a very reasonable suggestion (I know how silly that sounds :) ).

can you please jump start my poor brain with a couple more details on what you're looking for here?

Reply:
I meant something like this https://docs.docker.com/engine/reference/builder/#arg helps also give a view on the parametric nature of an entity in this case the Dockerfile itself